app/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
 
 
     defaultConfig {
         applicationId "au.com.bluedot.simpleintegration"
-        minSdkVersion 16
-        targetSdkVersion 29
+        minSdkVersion 21
+        targetSdkVersion 33
+        compileSdkVersion 33
```

Comment:
targeting 33 will need to implement runtime notification permission, have you implemented that?

Reply:
Yes, I have had the notification request implemented